posts/2020-06-03-asynchronous-programming-made-resilient-with-mpft.adoc

```diff
@@ -53,7 +53,10 @@ Usually when a method is called, it returns a result. However, when the `@Asynch
 image::/img/blog/FT-basic-asynchronous-execution.png[Fault Tolerance asynchronous execution flow]
 
 Sometime later, the method runs and returns a `CompletionStage` or `Future`.
-Now, Fault Tolerance completes the `CompletionStage` or `Future` that was returned earlier, with the value that was returned from the method. You might be wondering, *_"What if the returned result is an exception? What can I do?"_*. Read along.
+
+If the method returned a `Future`, then the last step is for Fault Tolerance to update the `Future` it returned to the user earlier so that it delegates to the `Future` returned from the method, allowing the user to see the result from the method.
```

Comment:
Suggestion

>If the method returns a `Future`, then the last step is for Fault Tolerance to update the `Future`  it previously returned to the user so that it delegates to the `Future` returned from the method, allowing the user to see the result from the method.